docs/getting-started-alvearie.md

```diff
@@ -23,7 +21,7 @@ To compile the De-Identification server, clone the GIT repository and compile wi
  - To start the server, use the following command: 
 
  ```
-   java -jar de-identification-app/target/de-identification-app-${RELEASE}-exec.jar
```

Comment:
The RELEASE environment variable does not "carry over" to the generated artifact names. Those are driven by the revision property.  I created a new section below which maintains the spirit of what we're striving for above.